evm/chain.py

```diff
@@ -130,8 +130,8 @@ def get_vm_class_for_block_number(self, block_number):
         for n in reversed(self.vms_by_range.keys()):
             if block_number >= n:
                 return self.vms_by_range[n]
-        raise EVMNotFound(
-            "There is no EVM available for block #{0}".format(block_number))
+        raise ChainNotFound(
```

Comment:
@pipermerriam Okay good to know, I'll keep that in mind and make another PR that fixes it soon